packages/form-js-viewer/src/util/index.js

```diff
@@ -60,4 +60,43 @@ export function generateIdForType(type) {
  */
 export function clone(data, replacer) {
   return JSON.parse(JSON.stringify(data, replacer));
+}
+
+/**
+ * Parse the schema for input variables a form might make use of
+ *
+ * @param {any} schema
+ *
+ * @return {string[]}
+ */
+
+export function getSchemaVariables(schema) {
+
+  if (!schema.components) {
+    return [];
+  }
+
+  return schema.components.reduce((variables, component) => {
+
+    const {
+      key,
+      valuesKey,
+      type
+    } = component;
+
+    if ([ 'text', 'button' ].includes(type)) {
```

Comment:
Would it be also an option to make this dependent on the form field implementation (as we do it [with sanitization](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js-viewer/src/import/Importer.js#L157)). This way we wouldn't have to touch this file once we add new form field types.

Reply:
That's not necessary, but maybe we could consider it.

Reply:
It does make sense, although I think this then involves a lot more of the code base for a relatively self contained utility, and a lot more configuration. I guess it depends how much flexibility we want to offer, but perhaps for now we can leave it at that until we find it restricts practical use cases. 

What do you think?

Reply:
Agree, let's keep it simple for now 👍